.github/workflows/ci.yml

```diff
@@ -22,17 +22,21 @@ jobs:
     - name: Run style and typing checks
       run: tox -e lint
   test:
-    runs-on: ${{ matrix.platform }}
+    runs-on: ${{ matrix.config.platform }}
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
-        platform: [ubuntu-latest]
+        config:
+          - {platform: ubuntu-latest, python-version: 3.6}
+          - {platform: ubuntu-latest, python-version: 3.7}
+          - {platform: ubuntu-latest, python-version: 3.8}
+          - {platform: ubuntu-latest, python-version: 3.9}
+          - {platform: windows-latest, python-version: 3.9}
```

Comment:
I think it's fine to only test the latest stabe python on Windows, because win users have to download/install it anyway so they end up using whatever is the latest. Also I wouldn't want to slow down the CI too much.